File: Score of a String/ScoreOfAString.scala
object Solution {
    def scoreOfString(s: String): Int = {
        var sum = 0;
        for (i <- 1 to s.length()-1) {
            sum += (s.charAt(i).toInt - s.charAt(i-1).toInt).abs
        }
        sum
    }
}

File: Score of a String/ScoreOfAString.js
/**
 * @param {string} s
 * @return {number}
 */
var scoreOfString = function (s) {
    var sum = 0;
    for (let i = 1; i < s.length; i++) {
        sum += Math.abs(s[i].charCodeAt(0) - s[i - 1].charCodeAt(0));
    }
    return sum;
};